site/demos/ecommerce-2/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
@SavvyShah Please add an appropriate readme.

The readme may have
- What and why of the application
- the live link of the application
- the preview image
- blog link